ament_cmake_test/ament_cmake_test-extras.cmake

```diff
@@ -18,10 +18,9 @@ enable_testing()
 # same option as in the CTest module
 option(BUILD_TESTING "Build the testing tree." ON)
 
-# option()
```

Comment:
Please leave this comment which indicates that this acts as an option.

Reply:
? It’s not. An option is a binary build flag. It’s a cache variable.

Reply:
I understand the technical difference. There are a few other comments identify CMake or environment variables a user can set to configure the build - all are annotated with the same comment like this one. So please keep it unchanged since it help finding those.

Reply:
Reverting. I still think it's wrong to have commented out "code", especially if that code (1) is broken and (2) implies something false. Maybe a standard comment header for options and cache variables would be better.